pylmod/tests/test_gradebook.py

```diff
@@ -17,6 +18,7 @@
 from pylmod.tests.common import BaseTest
 
 
+@ddt
```

Comment:
`@ddt.ddt` looks frequently used in edx though I didn't do an exhaustive search. If I remove this line I get test errors like:

    args = (<pylmod.tests.test_gradebook.TestGradebook testMethod=test_use_max_points_column>,)
    kw = {}
    
        @functools.wraps(test)
        def wrapper(*args, **kw):
            with httprettized():
    >           return test(*args, **kw)
    E           TypeError: test_use_max_points_column() takes exactly 5 arguments (1 given)